docs/stages.adoc

```diff
@@ -110,6 +110,64 @@ public SELF comma() {
 }
 ----
 
+=== Filler Words
+
+Filler words can be used to build a common and reusable vocabulary for your tests. This enables you to write
+beautifully fluent tests whilst at the same time keeping your stage methods clear and concise.
+
+The `@FillerWord` annotation can also be added to stage methods where you want to maintain the continuation of
+of sentences in reports.
+
+For example, given the following filler words:
+
+[source,java]
+----
+include::{sourcedir}/userguide/ExtendedVocabularyTest.java[tags=fillerWords,indent=0]
+----
+
+You can write test stages such as:
+
+[source,java]
+----
+include::{sourcedir}/userguide/ExtendedVocabularyTest.java[tags=givenFiller,indent=0]
+----
+
+This would generate the following report:
+
+----
+Given the ingredients
+      an egg
```

Comment:
question: If newlines are produced by `@IntroWord` and `an()` is not an intro word, how do we get this sentence on its own line?

Reply:
`@IntroWord` does not create new lines, it capitalises the word and adds it to the start of the sentence.

In this instance the new lines are created by the `ingredients` and `egg` stage methods.